upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.16.yaml.template

```
@@ -3924,6 +3924,12 @@ spec:
             # Enable WireGuard encryption for all on-the-wire pod-to-pod traffic
             - name: FELIX_WIREGUARDENABLED
               value: "{{ .Networking.Calico.WireguardEnabled }}"
+{{- if .EgressProxy }}
+{{ range $name, $value := ProxyEnv }}
+            - name: {{ $name }}
+              value: {{ $value }}
+{{ end }}
+{{- end }}
```

There is no need to check for `EgressProxy`, `ProxyEnv` already does that.
```suggestion
            # Enable support for HTTP forward proxy
            {{ range $name, $value := ProxyEnv }}
            - name: {{ $name }}
              value: {{ $value }}
            {{ end }}
```